pkg/resources/fluentbit/psp.go

```diff
@@ -25,6 +25,23 @@ import (
 
 func (r *Reconciler) clusterPodSecurityPolicy() (runtime.Object, reconciler.DesiredState, error) {
 	if r.Logging.Spec.FluentbitSpec.Security.PodSecurityPolicyCreate {
+		allowedHostPaths := []policyv1beta1.AllowedHostPath{{
+			PathPrefix: r.Logging.Spec.FluentbitSpec.MountPath,
+			ReadOnly:   true,
+		}, {
+			PathPrefix: "/var/log",
+			ReadOnly:   true,
+		}}
+
+		for _, vMnt := range r.Logging.Spec.FluentbitSpec.ExtraVolumeMount {
+			if vMnt.Source != "" && vMnt.Destination != "" {
```

Comment:
same as above, these fields shouldn't be empty

Reply:
you are right.